docs/source/docs/examples/poseest.md

```diff
@@ -0,0 +1,206 @@
+         :lines: 68-68
+         :linenos:
+         :lineno-start: 68
+
+    .. tab-item:: C++
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-cpp-examples/poseest/src/main/include/Constants.h
+         :language: c++
+         :lines: 42-43
+         :linenos:
+         :lineno-start: 42
+
+    .. tab-item:: Python
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-python-examples/poseest/robot.py
+         :language: python
+         :lines: 46-46
+         :linenos:
+         :lineno-start: 46
+
+```
+
+
+
+To incorporate PhotonVision, we need to create a {code}`PhotonCamera`:
+
+
+```{eval-rst}
+.. tab-set::
+
+    .. tab-item:: Java
+       :sync: java
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-java-examples/poseest/src/main/java/frc/robot/Vision.java
+         :language: java
+         :lines: 57-57
+         :linenos:
+         :lineno-start: 57
+
+    .. tab-item:: C++
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-cpp-examples/poseest/src/main/include/Vision.h
+         :language: c++
+         :lines: 145-145
+         :linenos:
+         :lineno-start: 145
+
+    .. tab-item:: Python
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-python-examples/poseest/robot.py
+         :language: python
+         :lines: 44-44
+         :linenos:
+         :lineno-start: 44
+```
+
+During periodic execution, we read back camera results. If we see AprilTags in the image, we calculate the camera-measured pose of the robot and pass it to the {code}`Drivetrain`.
+
+```{eval-rst}
+.. tab-set::
+
+    .. tab-item:: Java
+       :sync: java
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-java-examples/poseest/src/main/java/frc/robot/Robot.java
+         :language: java
+         :lines: 64-74
+         :linenos:
+         :lineno-start: 64
+
+    .. tab-item:: C++
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-cpp-examples/poseest/src/main/cpp/Robot.cpp
+         :language: c++
+         :lines: 38-46
+         :linenos:
+         :lineno-start: 38
+
+    .. tab-item:: Python
+
+       .. rli:: https://raw.githubusercontent.com/gerth2/photonvision/adb3098fbe0cdbc1a378c6d5a41126dd1d6d6955/photonlib-python-examples/poseest/robot.py
+         :language: python
+         :lines: 54-56
+         :linenos:
+         :lineno-start: 54
+
+```
+
+### Simulating the Camera
+
+First, we create a new {code}`VisionSystemSim` to represent our camera and coprocessor running PhotonVision, and moving around our simulated field.
```

Comment:
Presented without context it's unclear where I should put these two code snippets. How can we make it either link more easily back to the example project or show more context?

Reply:
Added some more comments and context. Short answer is I'm actually not sure I have an opinionated answer as to where to put it (Casserole will do it differently than the examples, but both are valid I think...). I'll put some advice to "keep like with like" and "see the full source code for more context"